README.md

```diff
@@ -4,16 +4,114 @@
 The golang client for CeresDB.
 CeresDB is a high-performance, distributed, schema-less, cloud native time-series database that can handle both time-series and analytics workloads.
 
-# Support features
-- [x] Query
-- [x] Write
+## Support features
+- Manage
+  - creating and alter table
+- Query data
+  - using sql
+- Write data
+  - Point build
 
-# Contributing
+## How To Use
```

Comment:
I prefer to add this usage to docs, and link there, otherwise we will have to maintain two copies

Reply:
The content if copy from docs, i think the readme also needs a simple example.
but according to your suggestion, we can add a link here to jump to doc, keep readme simple

Reply:
add link with docs, how about doc github link or https://docs.ceresdb.io/?